girder/web_client/src/misc.js

```diff
@@ -1,9 +1,12 @@
 import $ from 'jquery';
 import _ from 'underscore';
-import { Remarkable } from 'remarkable';
 
 import { MONTHS } from '@girder/core/constants';
 
+import { Remarkable } from 'remarkable';
```

Comment:
Is there a reason you moved these down? Third party package imports tend to be clustered together at the top, though we don't have a style rule to enforce this.

Reply:
My linter complained when I didn't move them down.  I'm not sure why -- it should be using the rules we have specified.